scripts/testing/dependency_solver.py

```diff
@@ -36,6 +36,11 @@
                      "python3-mock", "python3-nose-testconfig", "python3-sphinx_rtd_theme",
                      "python3-lxml", "python3-pip", "python3-coverage",
 
+                     # pep8 check
+                     "python-pycodestyle",
```

Comment:
Maybe we could install the pypi package instead of the rpm package to avoid issues we had with pylint.